api_callable.go

```diff
@@ -0,0 +1,98 @@
+package gax
+
+import (
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"time"
+)
+
+var (
+	retryableGrpcCodes []codes.Code = []codes.Code{
+		codes.DeadlineExceeded,
+		codes.Aborted,
+	}
+)
+
+// Represents a GRPC call stub.
+type APICall func(context.Context, interface{}) (interface{}, error)
+
+// isRetryable returns true if c represents a retryable GRPC code.
+func isRetryable(c codes.Code) bool {
+	for _, code := range retryableGrpcCodes {
```

Comment:
if you make retryableGrpcCodes a map, then this function body becomes a two-liner and lookup O(1)